on ln. 29, +0.0
	of __init__(...)
	| bound successfully to localhost:49930
on ln. 41, +5.33
	of handle_connections(...)
	| received connection from 127.0.0.1
on ln. 159, +7.34
	of handle_http_requests(...)
	| [127.0.0.1:37366] received 'GET /regis'
on ln. 41, +7.82
	of handle_connections(...)
	| received connection from 127.0.0.1
on ln. 159, +9.82
	of handle_http_requests(...)
	| [127.0.0.1:37370] received 'GET /favic'
on ln. 41, +11.33
	of handle_connections(...)
	| received connection from 127.0.0.1
on ln. 159, +13.34
	of handle_http_requests(...)
	| [127.0.0.1:37374] received 'POST /regi'
on ln. 41, +13.62
	of handle_connections(...)
	| received connection from 127.0.0.1
on ln. 159, +15.62
	of handle_http_requests(...)
	| [127.0.0.1:37378] received 'GET /favic'
on ln. 50, +31.68
	of handle_connections(...)
	| user-interrupt caught
	| fatal error, exiting ...
on ln. 59, +31.69
	of __del__(...)
	| garbage collecting SocketServer instance
